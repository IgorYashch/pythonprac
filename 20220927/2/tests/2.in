12,41,78,21,44,51
