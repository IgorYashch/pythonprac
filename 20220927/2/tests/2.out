[51, 44, 21, 12, 41, 78]
